packages/preview/mass-spec/0.1.0/typst.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "mass-spec"
+version = "0.1.0"
+entrypoint = "mass-spec.typ"
+authors = ["James (Fuzzy) Swift"]
+license = "MIT"
+description = "Mass spectra with annotations for typst."
+exclude = ["/assets/*"]
```

Comment:
The manual should also be excluded here (in particular the PDF) to keep the package size down.

Reply:
Will do